src/style.css

```diff
@@ -1 +1,93 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc') format('truetype');
+  font-weight: 400;
+  font-style: normal;
+}
+
+.form {
+  font-family: Avenir, sans-serif;
+  display: block;
+  margin-top: 20px;
```

Comment:
Be consistent with your margins. It is recommended to add only top or bottom margin, but not both, to avoid potential margin collapse issues.